### ISSUE TITLE
Display "what's new" popup the first time a user updates to new version

### ISSUE THREAD

**Issue by misaochan** [commons-app/apps-android-commons#1586]
Most people don't read changelogs on the Play Store, especially if their apps are set to auto-update. Some apps get around this by having a "What's new?" popup the first time a user updates, displaying a summary of changes that users are likely to find interesting.

Thoughts on us doing this? Pros: Some of our new features aren't very easy to discover, so having them all in one place might increase adoption of new features by users. Cons: Some people might find it annoying, even if it only pops up once.

**Comment by nicolas-raoul** [comment]
How about doing it only when really needed?
In particular, not for bugfix releases.

**Comment by misaochan** [comment]
Yeah, it should definitely be only done for major releases, if we go with this.

**Comment by neslihanturan** [comment]
We already have this now, I guess?

**Comment by chrisdebian** [comment]
This is done really well in [StreetComplete](https://streetcomplete.app/?lang=en-gb), it's a pop up on first use after a new release is installed, and is always available in the main menu:

![Screenshot_20241222-011653](https://github.com/user-attachments/assets/17627a62-77c1-40a9-b1f0-ffb84c2677f5)
